src/applications/appeals/995/tests/utils/submit.unit.spec.js

```diff
@@ -199,6 +202,21 @@ describe('getTimeZone', () => {
   });
 });
 
+describe('getClaimantData', () => {
+  // "other" types are not implemented, but there is some minimal code in place
+  it('should handle "other" claimant types', () => {
+    expect(
+      getClaimantData({
+        claimantType: 'other',
+        claimantTypeOtherValue: 'Twenty-five characters max',
+      }),
+    ).to.deep.equal({
+      claimantType: 'other',
+      claimantTypeOtherValue: 'Twenty-five characters ma',
```

Comment:
Should value here also be `Twenty-five characters max`?
Looks like the last letter may have been left off

Reply:
the `x` is in the 26th place, so it is expected to be removed.

Reply:
oh haha so this is on purpose